crates/amalthea/src/comm/ui_comm.rs

```diff
@@ -251,6 +251,25 @@ pub struct ShowUrlParams {
 	pub url: String,
 }
 
+/// Parameters for the ShowHtmlFile method.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct ShowHtmlFileParams {
+	/// The fully qualified filesystem path to the HTML file to display
+	pub path: String,
+
+	/// A title to be displayed in the viewer. May be empty, and can be
+	/// superseded by the title in the HTML file.
+	pub title: String,
```

Comment:
Could be `Option<String>` based on your docs here, but I guess you're looking for an empty string on the frontend side?

Reply:
A limitation of the code generator -- it supports optional properties on object, but not optional parameters on methods (which is where this structure is derived from).